src/main.rs

```diff
@@ -33,29 +35,40 @@ use crate::{
     util::json,
 };
 
-fn start_logger(default_level: LevelFilter) {
+fn start_logger(default_level: LevelFilter, with_syslog: bool) {
     let filter = match EnvFilter::try_from_default_env() {
-        Ok(filter) => filter
-            .add_directive("hyper=off".parse().unwrap())
-            .add_directive("ethers=off".parse().unwrap()),
-        _ => EnvFilter::default()
-            .add_directive(default_level.into())
-            .add_directive("hyper=off".parse().unwrap())
-            .add_directive("ethers=off".parse().unwrap()),
+        Ok(filter) => filter,
+        _ => EnvFilter::default().add_directive(default_level.into()),
     };
-
-    tracing_subscriber::fmt()
-        .with_env_filter(filter)
-        .with_target(false)
-        .init();
+    let filter = filter
+        .add_directive("hyper=off".parse().unwrap())
+        .add_directive("ethers=off".parse().unwrap());
+
+    let default_layer = tracing_subscriber::fmt::layer().with_target(false);
+
+    let subscriber = Registry::default().with(filter).with(default_layer);
+
+    if with_syslog {
+        let identity = CStr::from_bytes_with_nul(b"zksync-sr\0").unwrap();
+        let (options, facility) = Default::default();
+        let syslog = Syslog::new(identity, options, facility).unwrap();
+        let syslog_layer = tracing_subscriber::fmt::layer()
+            .with_target(false)
+            .with_ansi(false)
+            .without_time() // syslog logs its own time
```

Comment:
```suggestion
            .without_time() // Syslog logs its own time.
```
Just to be consistent with formatting :)